tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationBackgroundThreadTest.java

```diff
@@ -0,0 +1,906 @@
+
+    @Disabled("JDK-8349091") // FIX
+    @Test
+    public void areaChart() {
+        test(() -> {
+            AreaChart c = new AreaChart(createNumberAxis("x"), createNumberAxis("y"));
+            c.getData().setAll(createNumberSeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createNumberSeries());
+            accessChart(c);
+        });
+    }
+
+    @Disabled("JDK-8349091") // FIX
+    @Test
+    public void barChart() {
+        test(() -> {
+            BarChart c = new BarChart(createCategoryAxis("x"), createNumberAxis("y"));
+            c.getData().setAll(createCategorySeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createCategorySeries());
+            accessChart(c);
+        });
+    }
+
+    @Disabled("JDK-8349091") // FIX
+    @Test
+    public void bubbleChart() {
+        test(() -> {
+            BubbleChart c = new BubbleChart(createNumberAxis("x"), createNumberAxis("y"));
+            c.getData().setAll(createNumberSeries());
+            return c;
+        }, (c) -> {
+            c.getData().setAll(createNumberSeries());
+            accessChart(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void button() {
+        test(() -> {
+            Button c = new Button();
+            c.setSkin(new ButtonSkin(c));
+            return c;
+        }, (c) -> {
+            accessControl(c);
+            c.setAlignment(Pos.CENTER);
+            c.setText(nextString());
+            c.setDefaultButton(true);
+        });
+    }
+
+    @Test
+    public void canvas() {
+        test(() -> {
+            return new Canvas(30, 30);
+        }, (c) -> {
+            accessNode(c);
+            GraphicsContext g = c.getGraphicsContext2D();
+            g.setFill(Color.RED);
+            g.setStroke(Color.BLACK);
+            g.fillRect(5, 5, 5, 5);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void checkBox() {
+        test(() -> {
+            CheckBox c = new CheckBox("checkbox");
+            c.setSkin(new CheckBoxSkin(c));
+            return c;
+        }, (c) -> {
+            c.setAllowIndeterminate(true);
+            c.setSelected(true);
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void choiceBox() {
+        test(() -> {
+            ChoiceBox c = new ChoiceBox();
+            c.setSkin(new ChoiceBoxSkin(c));
+            return c;
+        }, (c) -> {
+            c.getItems().setAll("ChoiceBox", "1", "2");
+            c.getSelectionModel().select(0);
+            accessControl(c);
+        });
+    }
+
+    @Test
+    public void colorPicker() {
+        test(() -> {
+            ColorPicker c = new ColorPicker();
+            c.setSkin(new ColorPickerSkin(c));
+            c.setValue(Color.GREEN);
+            return c;
+        }, (c) -> {
+            c.show(); // does not fail here, unlike DatePicker?
```

Comment:
Interesting. I still think we should disallow it as I mentioned in the `DatePicker` case.